### ISSUE TITLE
Assigning admin role to other team members

### ISSUE THREAD

**Issue by alexandrugheorghe** [taskforcesh/issues#68]
Currently the account creator seems to have some kind of admin privileges that can't be handed over to other members of the team. This can cause an account to become unmanageable in the event that the initial account creator has to depart the company.

**Comment by manast** [comment]
Yes, currently there is no concept of Admin, it is just that the person that owns the subscription is the only one capable of creating organizations and teams, but for sure a better role management would be a great addition.

**Comment by manast** [comment]
It is now possible to have different roles in teams, however we do not yet support different roles within an organisation.
